### PR TITLE
Created new tabs and cards

### PR DESCRIPTION
**Pull request by cole-liferay**
### Description
Please explain the changes you made.  Include any relevant motivation, context and JIRA tickets.

### Checklist
- [ ] Code compiles without errors.
- [ ] Include tests for new features and functionality.
- [ ] Update README/documentation, when applicable.
- [ ] All tests, new and existing, pass without errors.
- [ ] Checked browser compatibility, especially IE11.

### CODE REVIEW

File: theme/src/resources/templates/category_pages/liferay_dxp_7_3.hbs [hunk 1]
```diff
@@ -28,9 +28,12 @@
 
 		const documentElementLang = document.documentElement.lang;
 
+		var alertChildren = null;
+
 		Liferay.render(
 			Liferay.ProductTabs,
 			{
+				alert: alertChildren,
```
Comment (rdai10) [inline theme/src/resources/templates/category_pages/liferay_dxp_7_3.hbs:36]: @cole-liferay 
If `alertChildren` is set as `null`, I don't think we need to include the alert here.

**Review by rdai10 — CHANGES_REQUESTED** [review]
@cole-liferay ,

Just a minor change request. Can you double check my suggestion and see if it can be removed?

**Review by cole-liferay — COMMENTED** [review]
Removed alert children references

**Review by rdai10 — APPROVED** [review]
Looks good, thanks @cole-liferay !

**Pull request merged**